app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/agenda_screen/AgendaScreen.kt

```diff
@@ -540,222 +557,214 @@ fun AgendaScreenContent(
                 .tinyHeight()
         )
 
+        // • EMPTY AGENDA INDICATOR
+        val showEmptyIndicator = remember { mutableStateOf(false) }
+        LaunchedEffect(agendaItems.isEmpty()) {
+            showEmptyIndicator.value = false
+
+            if (agendaItems.isEmpty()) {
+                delay(500)
```

Comment:
Why the delay? If it's not very obvious, I usually explain these things in a comment above to quickly understand them when coming back half a year later

Reply:
ok will add comment